### PR TITLE
Add Assets and Add Feature #3

### PR DESCRIPTION
**Pull request by Vanethos**
Added Feature #3 
Added assets to project, and SVG library.
Added correct type casting for lists in Middleware.

### CODE REVIEW

File: lib/screens/assets/images.dart [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const imgSvgFireman = "assets/fireman.svg";
+const imgSvgIconAlarm = "assets/ico_alarm.svg";
+const imgSvgIconFire = "assets/ico_fire.svg";
+const imgSvgIconWatch = "assets/ico_watch.svg";
```
Comment (guivazcabral) [inline lib/screens/assets/images.dart]: can you place all the icons under an `icons/` folder?

File: lib/middleware/fires_middleware.dart [hunk 2]
```diff
@@ -25,14 +26,100 @@ Middleware<AppState> _createLoadFires() {
...
+  var manFactor = 1.0;
+  var terrainFactor = 3.0;
+  var aerialFactor = 7.0;
+
+  var importance = data.human * manFactor + data.terrain * terrainFactor +
+      data.aerial * aerialFactor;
+
+  status[numberOfFires] += 1;
+  if (status[topImportance] < importance) {
+    status[topImportance] = importance;
```
Comment (guivazcabral) [inline lib/middleware/fires_middleware.dart]: this looks good, but i feel like these functions should not be mixed with middleware stuff. we should try to keep things tidy and clean. maybe create a separate file with these?

File: lib/screens/home_page.dart [hunk 3]
```diff
@@ -1,5 +1,8 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_redux/flutter_redux.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:fogosmobile/models/fire.dart';
```
Comment (guivazcabral) [inline lib/screens/home_page.dart]: this import is not being used

Comment (guivazcabral) [reply lib/screens/home_page.dart]: the fontawesome import is not being used too

File: lib/screens/assets/images.dart (hunk 1 above)
Comment (Vanethos) [reply lib/screens/assets/images.dart]: I'll create a new file under `assets` called `icons.dart`.

File: lib/middleware/fires_middleware.dart (hunk 2 above)
Comment (Vanethos) [reply lib/middleware/fires_middleware.dart]: created an `utils` folder

File: lib/screens/assets/images.dart (hunk 1 above)
Comment (Vanethos) [reply lib/screens/assets/images.dart]: But a quick question, do you want the `const` statements in a separate file or the actual svg files in a different folder at the root?
So the root would have
```
- assets
|___ icons
```

Comment (guivazcabral) [reply lib/screens/assets/images.dart]: sorry i meant the actual files

Comment (Vanethos) [reply lib/screens/assets/images.dart]: Added the correct folder for assets in the latest commit

**Pull request merged**